interfaces/acados_template/acados_template/c_templates_tera/acados_solver.in.c

```diff
@@ -33,6 +33,7 @@
 
 #include "acados_solver_{{ model.name }}.h"
 
+{# TODO: why is that logic needed? suggest to use "if (corresponding dim = 0 in renderer logic)" #}
```

Comment:
This is needed for MVSC compatibility, I thought it would be more transparent to have the machinery as a set of preprocessor instructions.